LiskHQ/lisk-desktop
# Unable to decrease the number of members of a multisig account

### What was the problem?

This PR resolves #5471

### How was it solved?

:bug: Fix remaining member signature calculation to use the correct keys

### How was it tested?

- Create a multisig account with 3 members
- Edit multisig account with 3 members to 2
- Also try to increase & decrease the members multiple times
- Send token transfer from updated account to cross check if its successful
